src/maker.rs

```diff
@@ -746,3 +746,127 @@ mod tests {
         assert_eq!(new_buy_order.quote.amount, dai_amount(18.0));
     }
 }
+
+#[cfg(test)]
+mod integration_tests {
```

Comment:
:upside_down_face: This doesn't look like an integration test to me :stuck_out_tongue:

Reply:
In general, I don't see the difference between this test and the ones above, other than the number of assertions.

Reply:
Yeah, that's why I stopped writing the test.
I don't think the test adds more value than the more fine-granular tests above, besides the fact that it showcases Nectar.